### DISCUSSION TITLE
Will consumers actually consume SBOMs (Homebrew question)

### DISCUSSION THREAD

**Discussion by sevansdell** [ossf/toolbelt#4] [General]
Just had a good interaction with WIlliam Woodruff in the Securing Software Repos WG for the value of Homebrew generating SBOMs. Will anyone actually use them? And what publicly available docs talk about this? https://openssf.slack.com/archives/C034CBLMQ9G/p1709656887381889

I would like to add this to our SBOM discussion Big picture. Where should we update this in the Goosatron layers?

**Comment by ware** [comment]
Hey Sarah.  Is this an area where the SBOM-Everywhere SIG can help?  @joshbressers may be able to help with this.
